src/main/java/uk/gov/justice/hmpps/prison/api/resource/OffenderSentenceResource.java

```diff
@@ -206,6 +206,7 @@ public List<OffenderSentenceDetail> postOffenderSentencesBookings(@RequestBody @
             @ApiResponse(responseCode = "404", description = "Requested resource not found.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))})})
     @Operation(summary = "Sentence term details for a prisoner")
     @GetMapping("/booking/{bookingId}/sentenceTerms")
+    @VerifyBookingAccess(overrideRoles = {"GLOBAL_SEARCH", "VIEW_PRISONER_DATA"})
```

Comment:
if new roles added to endpoint would be good to specify those in the `Operation` api docs

Reply:
you are right.  I keep forgetting to add them in